contracts/SimpleETHClaimer.sol

```diff
@@ -0,0 +1,134 @@
+
+    /// @notice Minimum initial expiration for a license.
+    uint256 public minClaimExpiration;
+
+    /// @notice Emitted when a parcel is claimed
+    event ParcelClaimed(uint256 indexed parcelId, address indexed owner);
+
+    constructor() {
+        _setupRole(DEFAULT_ADMIN_ROLE, msg.sender);
+        _setupRole(PAUSE_ROLE, msg.sender);
+    }
+
+    /**
+     * @notice Admin can update the minClaimExpiration.
+     * @param _minClaimExpiration The new minimum initial expiration for a license
+     * @custom:requires DEFAULT_ADMIN_ROLE
+     */
+    function setMinClaimExpiration(uint256 _minClaimExpiration)
+        external
+        onlyRole(DEFAULT_ADMIN_ROLE)
+    {
+        minClaimExpiration = _minClaimExpiration;
+    }
+
+    /**
+     * @notice Admin can update the license.
+     * @param licenseAddress The new license used to find owners
+     * @custom:requires DEFAULT_ADMIN_ROLE
+     */
+    function setLicense(address licenseAddress)
+        external
+        onlyRole(DEFAULT_ADMIN_ROLE)
+    {
+        license = ERC721License(licenseAddress);
+    }
+
+    /**
+     * @notice Admin can update the collector.
+     * @param collectorAddress The new collector
+     * @custom:requires DEFAULT_ADMIN_ROLE
+     */
+    function setCollector(address collectorAddress)
+        external
+        onlyRole(DEFAULT_ADMIN_ROLE)
+    {
+        collector = ETHExpirationCollector(collectorAddress);
+    }
+
+    /**
+     * @notice Admin can update the parcel.
+     * @param parcelAddress The new parcel
+     * @custom:requires DEFAULT_ADMIN_ROLE
+     */
+    function setParcel(address parcelAddress)
+        external
+        onlyRole(DEFAULT_ADMIN_ROLE)
+    {
+        parcel = GeoWebParcel(parcelAddress);
+    }
+
+    /**
+     * @notice Claim a new parcel with an initial contribution payment.
+     * @param to Address of license owner to be
+     * @param baseCoordinate Base coordinate of parcel to claim
+     * @param path Path of parcel to claim
+     * @param initialContributionRate Initial contribution rate of parcel
+     */
+    function claim(
+        address to,
+        uint64 baseCoordinate,
+        uint256[] calldata path,
+        uint256 initialContributionRate
+    ) external payable whenNotPaused {
+        // Build parcel
+        uint256 parcelId = parcel.build(baseCoordinate, path);
+
+        // Collect initial payment and set contribution rate
+        collector.setContributionRate{value: msg.value}(
+            parcelId,
+            initialContributionRate
+        );
+
+        uint256 expirationTimestamp = collector.licenseExpirationTimestamps(
+            parcelId
+        );
+
+        require(
+            (expirationTimestamp - block.timestamp) >= minClaimExpiration,
+            "Resulting expiration date must be at least minClaimExpiration"
+        );
+
+        emit ParcelClaimed(parcelId, to);
+
+        // Mint license last to avoid reentry
+        license.safeMint(to, parcelId);
+    }
```

Comment:
@codynhat i love how simple this contract is. I am assuming a few things here that may be off with my understanding of how this architecture works:

1) this contract is only used to register parcels that have **never** been claimed - this is why [maxId](https://github.com/Geo-Web-Project/core-contracts/pull/31/files#diff-0b3056cdcf850fb9cad3151cd9a34fcf1fd26cf3a1d41e43b9147d13aca3b9a3R78-R82) is being used to increment only `new` parcels
2) `ETHExpirationCollector#makePayment` is used after a parcel is claimed to keep the registration current (according to the contribution rate)
3) looks like `ETHPurchaser#purchase` is used on parcels that have already been claimed

are those assumptions correct?

if so it would seem like the dutch auction contract would use `SimpleETHClaimer#claim` in order to register an unregistered piece of land with no conflicting `path` attributes.

also, if you have time, can you provide some example values for these attributes:

`initialContributionRate` - have you all decided on a base contribution rate? i sort of forgot if this is a standard 
- value -- maybe @gravenp you have a reference point you can link me to?
- `path` - this comes in as an array of standard X and Y coordinates or is it converted to something else to comply with soldiity?
- `minClaimExpiration` - have you all decided on a minimum here yet?

Reply:
Those assumptions are correct.

A dutch auction could call `SimpleETHClaimer#claim`, but so can anyone else. I wrote this assuming that `SimpleETHClaimer` would be _replaced_ by a new dutch auction one. The new contract would call `GeoWebParcel#build` directly.

`initialContributionRate` is the contribution rate in wei per second for a particular parcel. So for a parcel with a value of 10 ETH and an annual tax rate of 10%, `initialContributionRate` would need to be:
```
10 * 0.1 = 1 ETH per year
1 ETH / (365 * 24 * 60 * 60) seconds in a year = 3.17e-8 ETH per second
3.17e-8 * 10^18 = 3.17e8 wei per second
```

For `path`, [this](https://github.com/Geo-Web-Project/specs/blob/main/contracts/GeoWebParcel.md#land-parcel-structure) may help.